_documentation/subscriptions/subscription-groups.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Subscription groups / tags / categories
+layout: doc
+level: "2"
+group: "subscriptions"
+icon: "fas fa-tags"
+---
+
+# Assigning groups / tags / categories to subscription
+
+To manage subscriptions it is possible to assign groups / tags / categories to a subscription.
+Please note that for the moment it is not possible to create queue(s) based on it. To do so it would first be needed to support multiple queues. (See related [issue 2648](https://github.com/AntennaPod/AntennaPod/issues/2648))
```

Comment:
I would put this in a separate paragraph (by introducing an extra white line. And some other changes:
```suggestion
Please note that for the moment it is not possible to create queues based on tags. This would require support for multiple queues first. (See also the [feature request](https://github.com/AntennaPod/AntennaPod/issues/2648) on our development platform.)
```